examples/node-groups/fargate-profiles/main.tf

```diff
@@ -91,7 +91,7 @@ module "aws_vpc" {
 # Example to consume aws-eks-accelerator-for-terraform module
 #---------------------------------------------------------------
 module "aws-eks-accelerator-for-terraform" {
-  source = "../.."
+  source = "github.com/aws-samples/aws-eks-accelerator-for-terraform"
```

Comment:
think we want to leave this as source. Otherwise running CI on these blueprints will pull from main, not the branch.